.circleci/config.yml

```diff
@@ -335,7 +335,32 @@ jobs:
           registry: registry.heroku.com
           tag: latest
       - run: heroku container:release -a simple-hub-staging simple-hub
-
+      
+  deploy-persistent-seeder:
+    working_directory: /home/circleci/project
+    resource_class: large
+    docker:
+      - image: circleci/node:10.16.3-browsers
+    steps:
+      - setup_remote_docker
+      - checkout
+      - <<: *restore_dep
+      - <<: *restore_lib
+      - run: rm -rf ./patches && cd ./packages/ && find . -maxdepth 1 ! -path "./channel-provider" ! -path "./client-api-schema" ! -path "./e2e-tests" ! -path "./devtools" -exec rm -rf {} ';'
+      - run: yarn --frozen-lockfile
+      - run: curl https://cli-assets.heroku.com/install.sh | sh
+      - run: heroku container:login
```

Comment:
Can any of these things be defined once and reused since they're partially similar to the hub deployment config?

Reply:
I'll take a look right now

Reply:
Done in https://github.com/statechannels/monorepo/pull/1647/commits/ec540f5477405ad9ec9ba8ee88dd6488016ea072 and https://github.com/statechannels/monorepo/pull/1647/commits/fe450db18bfb451b4c72f5eae1dbb54b2ce2f5e7